server/docs/source/index.html.md

```diff
@@ -286,15 +286,15 @@ follow | Stream logs
 
 ```json
 {
-	"id": "mygrid/misty-sun-87",
-	"node_id": "RQKP:Y32W:SB4H:7TNG:5BKC:R6ZO:5B25:C2AV:3Z3Q:SVPX:A76C:WPBX",
+	"id": "RQKP:Y32W:SB4H:7TNG:5BKC:R6ZO:5B25:C2AV:3Z3Q:SVPX:A76C:WPBX",
 	"name": "misty-sun-87",
 	"connected": "true",
-	"created_at": "",
-	"updated_at": "",
-	"last_seen_at": "",
+	"created_at": "2017-06-14T12:33:05.139Z",
+	"updated_at": "2017-06-14T13:37:36.968Z",
+	"last_seen_at": "2017-06-14T13:38:03.785Z",
+	"connected_at": "2017-06-14T12:33:05.084+00:00",
```

Comment:
Looks like a bug, should probably use the same `Z` UTC timezone suffis as the other timestamps... seems to be some odd `Time` vs `DateTime` difference.. :cry: